### PR TITLE
Left toolbar connect

### PR DESCRIPTION
**Pull request by luceestrong**
connected to store

### CODE REVIEW

File: front-end/src/components/UserInfoToolbar/UserInfoToolbar.jsx [hunk 1]
```diff
@@ -34,10 +35,11 @@ export default function UserInfoToolbar() {
     dispatch({type: 'SET_MODAL', payload: false});
     clearRecipe();
   };
-
+  // app, recipe page, user
```
Comment (k12kekeal) [inline front-end/src/components/UserInfoToolbar/UserInfoToolbar.jsx:38]: I think these might just be notes about where the router stuff is located? If you're not usin 'em anymore, I recommend deleting!

**Review by k12kekeal — APPROVED** [review]
Looks good, Lucy! Great job~

File: front-end/src/components/UserInfoToolbar/UserInfoToolbar.jsx [hunk 2]
```diff
@@ -34,10 +35,11 @@ export default function UserInfoToolbar() {
...
   if (user) {
-    const { name, picture, email, nickname } = user;
-    const userLink = `/user/${name}`;
+    const { name, picture, email, _id } = user;
+    // change to userID
+    const userLink = `/user/${_id}`;
```
Comment (chaomonica) [inline front-end/src/components/UserInfoToolbar/UserInfoToolbar.jsx:42]: is userLink used? XD

**Review by chaomonica — APPROVED** [review]

File: front-end/src/components/UserInfoToolbar/UserInfoToolbar.jsx (hunk 2 above)
Comment (k12kekeal) [reply front-end/src/components/UserInfoToolbar/UserInfoToolbar.jsx:42]: I think it's hidden by default in the review bc it wasn't part of the changes :P it gets used on line 46 + 49!

**Pull request merged**